{"uuid":"f07317c2-8c11-4340-907c-d8cb47edb3f5","historyId":"42d888d3da2121c8c51fa273089e10af","fullName":"tests.LoginTests.testUserRegistration","labels":[{"name":"package","value":"tests.LoginTests"},{"name":"testClass","value":"tests.LoginTests"},{"name":"testMethod","value":"testUserRegistration"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"TestNGTest-65d39a66-1071-4b9f-9671-99113105dd1d"},{"name":"subSuite","value":"tests.LoginTests"},{"name":"host","value":"mohamed-sami"},{"name":"thread","value":"25052@mohamed-sami.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"story","value":"User Registration"},{"name":"feature","value":"Login and Registration"},{"name":"epic","value":"User Management"},{"name":"severity","value":"critical"}],"links":[],"name":"testUserRegistration","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"User should be logged in after successful registration and login expected [true] but found [false]","trace":"java.lang.AssertionError: User should be logged in after successful registration and login expected [true] but found [false]\r\n\tat org.testng.Assert.fail(Assert.java:111)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1578)\r\n\tat org.testng.Assert.assertTrue(Assert.java:57)\r\n\tat tests.LoginTests.testUserRegistration(LoginTests.java:35)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1541)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:848)\r\n\tat org.testng.TestRunner.run(TestRunner.java:621)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:443)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:336)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1200)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1114)\r\n\tat org.testng.TestNG.run(TestNG.java:1082)\r\n\tat com.microsoft.java.test.runner.testng.TestNGRunner.run(TestNGRunner.java:45)\r\n\tat com.microsoft.java.test.runner.testng.TestNGLauncher.execute(TestNGLauncher.java:32)\r\n\tat com.microsoft.java.test.runner.Launcher.main(Launcher.java:57)\r\n"},"stage":"finished","description":"Test Case 1: Verify that a new user can successfully register with valid details","steps":[],"attachments":[],"parameters":[],"start":1748464235518,"stop":1748464246081}